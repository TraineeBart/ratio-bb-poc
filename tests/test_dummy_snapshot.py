def test_dummy_snapshot(snapshot):
    value = {"key": "value"}
    assert value == snapshot