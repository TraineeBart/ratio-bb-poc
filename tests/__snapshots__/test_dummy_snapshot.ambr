# serializer version: 1
# name: test_dummy_snapshot
  dict({
    'key': 'value',
  })
# ---
